core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -404,12 +414,14 @@ export class TranslationModalComponent {
       // img_2021029_210552_zbmdt94_height_54_width_490.png&amp;quot;">
       // </oppia-noninteractive-image>
       if (element.localName === 'oppia-noninteractive-image') {
-        const attribute = element.attributes[type].value;
-        return attribute.substring(
-          textWrapperLength, attribute.length - textWrapperLength);
+        const attribute = element.attributes[
+          type as keyof NamedNodeMap] as Attr;
+        const attributeValue = attribute.value;
```

Comment:
This separate variable declaration doesn't add much value (pun not intended). I would just inline it above.

Reply:
Not understand.

Reply:
I meant we can combine this statement with the above, but it's fine to leave as is.